Flashcards.Dejmenek/Flashcards.Dejmenek/Program.cs

```diff
@@ -0,0 +1,173 @@
+﻿using Flashcards.Dejmenek.Controllers;
+using Flashcards.Dejmenek.DataAccess;
+using Flashcards.Dejmenek.DataAccess.Repositories;
+using Flashcards.Dejmenek.Enums;
+using Flashcards.Dejmenek.Helpers;
+using Flashcards.Dejmenek.Services;
+using Spectre.Console;
+
+internal class Program
+{
```

Comment:
Can you extract the nested logic to their own methods in order to reduce nesting?